File: sdks/thunkmetrc-go/wrapper/models/location_type.go
package models

type LocationType struct {
    ForHarvests bool `json:"ForHarvests,omitempty"`
    ForPackages bool `json:"ForPackages,omitempty"`
    ForPlantBatches bool `json:"ForPlantBatches,omitempty"`
    ForPlants bool `json:"ForPlants,omitempty"`
    Id int64 `json:"Id,omitempty"`
    Name string `json:"Name,omitempty"`
}

File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/LocationType.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class LocationType(
    @JsonProperty("ForHarvests")
    val forHarvests: Boolean? = null,
    @JsonProperty("ForPackages")
    val forPackages: Boolean? = null,
    @JsonProperty("ForPlantBatches")
    val forPlantBatches: Boolean? = null,
    @JsonProperty("ForPlants")
    val forPlants: Boolean? = null,
    @JsonProperty("Id")
    val id: Int? = null,
    @JsonProperty("Name")
    val name: String? = null
)
